frontend/app/routes/_public+/apply+/$id+/type-of-application.tsx

```diff
@@ -0,0 +1,121 @@
+import { useEffect } from 'react';
+
+import { ActionFunctionArgs, LoaderFunctionArgs, json, redirect } from '@remix-run/node';
+import { Form, useActionData, useLoaderData } from '@remix-run/react';
+
+import { faChevronLeft, faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useTranslation } from 'react-i18next';
+import { z } from 'zod';
+
+import { Button, ButtonLink } from '~/components/buttons';
+import { ErrorSummary, createErrorSummaryItems, hasErrors, scrollAndFocusToErrorSummary } from '~/components/error-summary';
+import { InputRadios } from '~/components/input-radios';
+import { getApplyFlow } from '~/routes-flow/apply-flow';
+import { getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { RouteHandleData } from '~/utils/route-utils';
+
+const i18nNamespaces = getTypedI18nNamespaces('eligibility');
+
+export const handle = {
+  breadcrumbs: [
+    // prettier-ignore
+    { labelI18nKey: 'eligibility:breadcrumbs.canada-ca', to: '/personal-information' },
+    { labelI18nKey: 'eligibility:breadcrumbs.benefits' },
+    { labelI18nKey: 'eligibility:breadcrumbs.dental-coverage' },
+    { labelI18nKey: 'eligibility:breadcrumbs.canadian-dental-care-plan' },
+  ],
```

Comment:
**Suggestion:** The breadcrumbs for all these eligibility pages (and all pages of the CDCP) appear to be the same. Consider moving the breadcrumbs into the layout instead.

Reply:
Moved breadcrumbs to the application layout component

Reply:
Since the breadcrumbs in the protected and public pages are different, the breadcrumbs should be specified in the `/_public+/_layout.tsx`'s `handle` function.

Reply:
Removed from application layout and put it in public layout as the handle